scripts/deploy.sh

```diff
@@ -24,8 +24,12 @@ fi
 echo "Found device at $device_path"
 
 echo 'Copying application code...'
-rsync --verbose --recursive --delete --checksum \
+rsync --verbose --recursive --delete --checksum -@2 \
```

Comment:
What does the `2` do? Also prefer to use longhand notation for flags over shorthand in scripts for self-documentation.

Reply:
This improves detecting that a file is unchanged via timestamps, and speeds up rsync as a result (though it may also need --times so that the target timestamp is set to the source timestamp).

It should be -@1 or --modify-window=1, not 2. from `man rsync`:
```
       --modify-window=NUM, -@
              When comparing two timestamps, rsync treats  the  timestamps  as
              being  equal  if  they  differ by no more than the modify-window
              value.  The default is 0, which matches  just  integer  seconds.
              If  you  specify  a negative value (and the receiver is at least
              version 3.1.3) then nanoseconds will also be taken into account.
              Specifying  1  is  useful  for  copies  to/from  MS  Windows FAT
              filesystems, because FAT represents times with a 2-second  reso‐
              lution  (allowing  times  to differ from the original by up to 1
              second).

```

Reply:
Gotcha, please use `--modify-window` instead of `-@`